API-strategie-extensies/ext-geospatial.md

```diff
@@ -123,18 +123,48 @@ The guiding priciples for CRS support:
 - Exchange format (notation) RD and Web Mercator X Y in meters: `195427.5200 311611.8400`
 
 <div class="rule" id="api-40">
-  <p class="rulelab"><strong>API-40</strong>: Pass the coordinate reference system (CRS) of the request and the response in the headers</p>
-  <p>The coordinate reference system (CRS) for both the request and the response are passed as part of the request headers and reponse headers. In case this header is missing, send the HTTP status code <code>412 Precondition Failed</code>.</p>
+  <p class="rulelab"><strong>API-40</strong>: Pass the coordinate reference system (CRS) of the request and the desired CRS of the response as parameters</p>
+  <p>The coordinate reference system (CRS) for both the request and the response are passed as parameters. The CRS of the geometry in the response is specified using a header.</p>
 </div>
 
-The following headers are purely meant for negotiation between the client and the server. Depending on the application, the request not only contains geometries but also specific meta data, e.g. the original realistion including the collection date.
+This method conforms to the standard OGC API Features part 2: Coordinate Reference Systems by Reference. 
+
+Use the following parameters: 
+- A parameter `crs` to specify the CRS used in a request. The value of the parameter is a URI identifier of the CRS. If the `crs` parameter is specified, then the coordinates of all geometry-valued properties in the response document must be presented in the requested CRS. If the requested CRS is not available, respond with HTTP status code 400.
+- A parameter `bbox-crs` to assert the CRS used for the coordinate values, in case the `bbox` parameter is used in the request. 
```

Comment:
Zoals ook aangegeven tijdens het overleg van 24-2-2022 is een bbox-crs parameter een beetje beperkt. Als OGC API Features in een later stadium meer geometrie types gaat ondersteunen om mee te filteren, krijgen die dan allemaal een eigen xxx-crs parameter? We kennen nu al use cases waarbij ook andere geometrie types ondersteund moeten worden, dan lijkt het me goed om daar bij de NL API Strategie al rekening mee te houden. Een Content-Crs parameter die voor alle geometrie types gebruikt kan worden, lijkt me dan een toekomstbestendigere en herbruikbare oplossing.

Reply:
OGC API Features part 3 voegt [filter-crs](http://docs.ogc.org/DRAFTS/19-079r1.html#filter-filter-crs) toe. Ik stel toch voor om hiervan uit te gaan en niet een generiekere content-crs parameter te introduceren.

Reply:
Een filter-crs parameter lijkt mij een prima oplossing om aan te geven in welk CRS de geometrie in de request is.
Moet daarnaast dan ook de bbox-crs parameter ondersteund worden?
Welke parameter gebruiken om aan te geven in welk CRS de geometrie in de response moet zijn?
Zie ook https://github.com/Geonovum/KP-APIs/pull/434#discussion_r813741022 hierboven.

Reply:
`filter-crs` ga ik toevoegen.